### ISSUE TITLE
newly installed app shows in a language that seem to not exist 

### ISSUE THREAD

**Issue by DidiRamsaran** [JabRef/jabref#10212]
### JabRef version

5.9 (latest release)

### Operating system

Windows

### Details on version and operating system

Windows 11 home 22H2

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. ...I dont really know how to do it
2. ...
3. ...

### Appendix

...
<details>

<summary>Log File</summary>
![image](https://github.com/JabRef/jabref/assets/86619151/929ee175-f73b-400e-be17-34d9e91ef905)

```
Paste an excerpt of your log file here
```

</details>

**Comment by DidiRamsaran** [comment]
![image](https://github.com/JabRef/jabref/assets/86619151/15f87364-b57c-4b8b-a650-63553d5809f7)

**Comment by DidiRamsaran** [comment]
![image](https://github.com/JabRef/jabref/assets/86619151/d210a143-ea9b-4882-b3b8-9606a16c79dd)


so this is what it looks like

**Comment by Siedlerchr** [comment]
Could you solve it?

**Comment by DidiRamsaran** [comment]
no, i dont really know how

**Comment by Siedlerchr** [comment]
I remember that we had another user with this problem as well:
https://github.com/JabRef/jabref/issues/9423#issuecomment-1336439055 
Can you try if any of these work?

**Comment by papatekken** [comment]
Did the problem happen in the first run after installed?

**Comment by DidiRamsaran** [comment]
yes! I installed it and happens every time I open the app. I tried uninstalling and running the 5.8 version of the app, but still nothing changes.

My pc is in English (US) and the font I have is Segoe UI, i believe it's got something to do with that according to what ive searched and to Sied's comment above yours

**Comment by DidiRamsaran** [comment]
yes! I installed it and happens every time I open the app. I tried uninstalling and running the 5.8 version of the app, but still nothing changes.

My pc is in English (US) and the font I have is Segoe UI, i believe it's got something to do with that according to what ive searched and to Sied's comment above your's

**Comment by ThiloteE** [comment]
Labeled wontfix, as this very likely is an issue with fonts installed and managed by your operating system. Depends on external.

**Comment by jsmm** [comment]
Same problem here, and it happens only to JabRef, so I doubt it is a wider Windows problem.  
I am using JabRef 5.15 on Windows 11 Pro (22631.4169). According to System File Checker, Windows has no problems: I have run dism.exe to cleanup-image and sfc /scannow. 
I have reset fonts in the two languages I have installed: Spanish (Spain) and English (United States). The gibberish fonts in JabRef persist whether I use English or Spanish as Windows admin language.
I am using JabRef 5.15 on Windows 11 Pro (22631.4169).
